src/bufr/BufrParser/Exports/Variables/SensorScanAngleVariable.cpp

```diff
@@ -25,13 +25,13 @@ namespace
     namespace ConfKeys
     {
         const char* FieldOfViewNumber = "fieldOfViewNumber";
-        const char* HeightOfStation = "heightOfStation";
+//      const char* HeightOfStation = "heightOfStation";
```

Comment:
@emilyhcliu any reason why we're not just removing this line?  seems like lower down you are removing `HMSL` from `test/testinput/bufr_ncep_atms.yaml` for instance

Reply:
```suggestion
```